golem/apps/manager.py

```diff
@@ -10,9 +12,23 @@
 class AppManager:
     """ Manager class for applications using Task API. """
 
-    def __init__(self) -> None:
+    def __init__(self, app_dir: Path, save_apps=True) -> None:
         self._apps: Dict[AppId, AppDefinition] = {}
         self._state = AppStates()
+        self._app_file_names: Dict[AppId, Path] = dict()
+
+        # Save build in apps, then load apps from path
+        built_in_apps: List[AppId] = []
+        if save_apps:
+            built_in_apps = save_built_in_app_definitions(app_dir)
+        for (app_def_path, app_def) in load_apps_from_dir(app_dir):
```

Comment:
```suggestion
        for app_def_path, app_def in load_apps_from_dir(app_dir):
```
It's not an error, though.

Reply:
Fixed it, thanks!

Wanted to rename `build_in_apps` to `new_apps` since its more accurate, but did not want to rebuild for only this ;)